core/src/main/scala/org/apache/spark/util/Utils.scala

```diff
@@ -1872,6 +1872,12 @@ private[spark] object Utils extends Logging {
     }
   }
 
+  /** Returns true if the given exception is a linkage error. */
+  def isLinkageError(e: Throwable): Boolean = e match {
+    case _: java.lang.LinkageError => true
```

Comment:
This can just be `e.instanceOf[LinkageError]` but an entire method for this is not appropriate